release/prepareRelease.sh.cmake

```diff
@@ -238,39 +259,56 @@ case "$SYSTEM_NAME" in
 		lua=""
 		case "$SYSTEM_ARCH" in
 			i686|*86)
-				if [ -d /usr/i686-pc-mingw32 ]; then
+				if [ -d /usr/i686-w64-mingw32 ]; then
+					mingw=/usr/i686-w64-mingw32
+				elif [ -d /usr/i686-pc-mingw32 ]; then
 					mingw=/usr/i686-pc-mingw32
 				else
 					mingw=/usr/i686-mingw32
 				fi
 				lua=win32
 			;;
 			x86_64|*64)
-				if [ -d /usr/i686-pc-mingw32 ]; then
+				if [ -d /usr/i686-w64-mingw32 ]; then
+					mingw=/usr/x86_64-w64-mingw32
```

Comment:
Isn't this the path to the compiler? We can get it from there I guess. (see above)

Reply:
yep, CMAKE_FIND_ROOT_PATH ;-)